personal-practice/Dice.java

```diff
@@ -0,0 +1,23 @@
+import java.awt.*;
+import java.util.Scanner;
+import java.util.concurrent.Future;
+import java.lang.reflect.Array;
+import java.util.Arrays;
+import java.util.Random;
```

Comment:
sort import statements: https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing